argilla-server/tests/unit/api/handlers/v1/datasets/records/records_bulk/test_dataset_records_bulk.py

```diff
@@ -141,6 +141,32 @@ async def test_update_record_metadata_by_external_id(
         for record in updated_records:
             assert record.metadata_ == metadata
 
+    async def test_update_record_metadata_with_invalid_external_id_but_correct_id(
+        self, async_client: AsyncClient, db: AsyncSession, owner_auth_header: dict
+    ):
+        dataset = await self.test_dataset()
+        records = await RecordFactory.create_batch(dataset=dataset, size=100)
```

Comment:
Can we reduce `100` to a smaller number? These kind of things can increase the whole time spent running the test suite.